src/sentry/api/endpoints/sentry_app_installation_external_issue_details.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework.response import Response
+
+from sentry.api.bases import (
+    SentryAppInstallationExternalIssueBaseEndpoint as ExternalIssueBaseEndpoint,
+)
+from sentry.mediators import external_issues
+from sentry.models import PlatformExternalIssue
+
+
+class SentryAppInstallationExternalIssueDetailsEndpoint(ExternalIssueBaseEndpoint):
+    def delete(self, request, installation, external_issue_id):
+        try:
+            platform_external_issue = PlatformExternalIssue.objects.get(
+                id=external_issue_id,
```

Comment:
I think we should also add the `service_type` to the query here so we can make sure external issue actually belongs to the install